webpack/scenes/Subscriptions/__tests__/__snapshots__/SubscriptionsReducer.test.js.snap

```diff
@@ -668,7 +668,7 @@ Object {
   "disconnected": false,
   "itemCount": 0,
   "loading": true,
-  "manifestModalOpened": false,
+  "manifestModalOpened": true,
```

Comment:
Why is this changed? 
`Subscriptions reducer should return the initial state 1` but initial state in the reducer is `  manifestModalOpened: false,`

Reply:
@MariaAga `manifestModalOpened` was used to keep track of the old modal, so it's no longer needed.   I removed it, thanks